README.md

```diff
@@ -1,6 +1,32 @@
 # TODO
 
-* Write readme
+* Update readme
+
+# Getting Started
+
+## Start local development environment with Docker
+
+1. Start by copying the `.env.example` file into a new `.env` file:
+```
+cp .env.example .env
+```
+2. Create a private key:
+```
+make keys
+```
+3. Copy the private key contents into the `PRIVATE_KEY` environment variable within the `.env` file:
```

Comment:
Maybe describe what needs to be done? Replace actual newlines with \n

Reply:
Maybe we should even let the make keys command output: PRIVATE_KEY="..." ?

Reply:
>Maybe we should even let the make keys command output: PRIVATE_KEY="..." ?
I didn't do that because dumping the private key contents on the screen didn't seem like a good practice and copy/pasting from the terminal could also lead to unexpected newlines.